scenarioo-client/app/shared/utils/urlContextExtractor.service.spec.ts

```diff
@@ -0,0 +1,71 @@
+/* scenarioo-client
+ * Copyright (C) 2014, scenarioo.org Development Team
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {UrlContextExtractorService} from './urlContextExtractor.service';
+
+declare var angular: angular.IAngularStatic;
+
+describe('UrlContextExtractorService', () => {
+
+    let urlContextExtractorService: UrlContextExtractorService;
+
+    beforeEach(() => {
+        urlContextExtractorService = new UrlContextExtractorService();
+    });
+
+    it('returns empty string for usual URLs without context', () => {
+        void expect(urlContextExtractorService.getContextPathFromUrl('demo.scenarioo.org')).toBe('');
```

Comment:
'void' should not be needed in front of expect() as far as I know.

Reply:
Without 'void' I got the error message 'Promises must be handled appropriately' in the CircleCI JS tests.

Reply:
Ah, I didn't see that. In that case 'await' would probably be the better keyword.
See: https://jasmine.github.io/tutorials/async

Reply:
Thanks for the hint.